api/src/fs.ts

```diff
@@ -61,8 +61,8 @@ async function readTextFile(
   filePath: string,
   options: FsOptions = {}
 ): Promise<string> {
-  return await promisified<string>({
-    module: 'Fs',
+  return await invoke<string>({
```

Comment:
You don't need to await it if you are returning it.

Reply:
the await is added by the stupid linter

Reply:
Jeez is that eslint that's doing all that stuff?

Reply:
yeah, can you see what's going on? I hope that's not the default config, I hope we messed up :joy:

Reply:
Okay so it seems like there is a trade-off if you just `return` instead of `return await`. Just `return` has better performance because it creates less promises, but `return await` will show that function in the stack trace. However, due to some rollup stuff (I assume), the stack trace is the same no matter what. I opened #1245 to fix this.